debian-buster/miniconda3.dockerfile

```diff
@@ -0,0 +1,62 @@
+# Authors:
+#   Unai Martinez-Corral
```

Comment:
I feel you wrote this almost from scratch. No need to add me as an author unless you actually copy and paste some relevant portion of a source written by me.